war/src/main/scss/components/_badges.scss

```diff
@@ -31,3 +31,31 @@
 .am-badge {
   display: inline;
 }
+
+.badge {
```

Comment:
this is a bit too generic. Please see https://weekly.ci.jenkins.io/design-library/Conventions/ for conventions how to name classes.
This means it should be something like `.jenkins-badge`